webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityComposerDrawer.vue

```diff
@@ -43,7 +43,9 @@
             attachment-enabled
             autofocus />
         </v-card-text>
-        <attach-image-component :maxFileSize="$root.maxFileSize" />
+        <attach-image-component 
+          :maxFileSize="$root.maxFileSize"
+          @attachments-uploaded="canPostActivity = true" />
```

Comment:
we may need to disable posting until the images are uploaded or deleted, @SaraBoutej WDYT ?

```suggestion
          @attachments-uploading="canPostActivity = false" />
          @attachments-uploaded="canPostActivity = true" />
```

Reply:
@boubaker yes I changed that, until all file.progress === 100